app/views/posts/index.html.erb

```diff
@@ -26,7 +26,7 @@
           <div class="singleComment">
```

Comment:
- [ ] I'm not able to create a post. After filling up the title and the text fields and clicking on submit button, nothing happens. Please ensure your app works correctly (The user should be able to create posts, comments, and add likes). Make sure that the flash messages for successful and not successful paths are rendered in the view after creating post, comments, and likes.